examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -69,11 +69,11 @@
    "source": [
     "# Connect the Jupyter Notebook to the workers\n",
     "\n",
-    "The first step is to let the jupyter notebook connect to the workers (cluster). To achieve that, imports the ipypyarallel library since we rely on it to manage the workers, then connects it through `ipp.Client(profile=\"bluefog\")` as shown in the following cell.\n",
+    "The first step is to let the jupyter notebook connect to the workers (cluster). To achieve that, import the ipypyarallel library since we rely on it to manage the workers, then connect it through `ipp.Client(profile=\"bluefog\")` as shown in the following cell.\n",
     "\n",
-    "`rc` (remote client) class is the key that you will use to send the code from this notebook (driver) to the workers. Remote client is called the remote because they talk with the workers, which may not be on the local host.\n",
-    "To check the notebook has connected to the worker succesfully, you can look the ID of rc through `rc.ids`.\n",
-    "Depending on how many processes you starts, the length of rc will be the same number as the number of process you start the `ibfrun`, i.e., the number after `-np`."
+    "`rc` (remote client) class is the key that you will use to send the code from this notebook (driver) to the workers. Remote client is called as such because it talks with the workers, which may not be on the local host.\n",
+    "To check if the notebook has connected to the worker succesfully, you can look the ID of rc through `rc.ids`.\n",
+    "Depending on how many processes you starts, the length of rc will be the same number as the number of process you starts the `ibfrun`, i.e., the number after `-np`."
```

Comment:
Please change **starts** to **start**, not the other way around. 😂 
- Depending on how many processes you **starts** -> Depending on how many processes you **start**
- the number of process you **starts** the `ibfrun` -> the number of process you **start** the `ibfrun`